package.json

```
@@ -13,7 +13,7 @@
     "lint": "turbo run lint",
     "lint-staged": "lint-staged",
     "lint:fix": "turbo run lint:fix",
-    "lint:packages": "sherif -p crm -p invoices",
+    "lint:packages": "sherif -p crm",
```

```suggestion
    "lint:packages": "sherif",
```
As crm is also removed 😄